lib/vmpooler/api.rb

```diff
@@ -168,6 +168,45 @@ def hostname_shorten hostname
           JSON.pretty_generate(result)
         end
 
+        get '/status/?' do
+          content_type :json
+
+          result = {}
+          clone_times = []
+
+          result['capacity_current'] = 0
+          result['capacity_total'] = 0
+
+          result['status'] = 1
+
+          $config[:pools].each do |pool|
+            result['capacity_current'] += $redis.scard( 'vmpooler__ready__' + pool['name'] ).to_i
+            result['capacity_total'] += pool['size'].to_i
+
+            if ( $redis.scard( 'vmpooler__ready__' + pool['name'] ).to_i == 0 )
```

Comment:
Actually, in hindsight, 'current_capacity' is OVERALL throughout all of the pools.  This check is just to see if the existing pool is empty.